soc/realtek/ec/rts5912/reg/reg_uart.h

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (c) 2023 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _REALTEK_RTS5912_REG_UART_H
+#define _REALTEK_RTS5912_REG_UART_H
+
+/*
+ * @brief UART Controller (UART)
+ */
+
+typedef struct {
+	union {
+		union {
+			const volatile uint32_t RBR;
+			struct {
+				const volatile uint32_t DATA: 8;
+				uint32_t reserved1: 24;
+			} RBR_b;
```

Comment:
Are any of these bitfield used?  It looks like all register read/writes use the `uint32_t` register names and use the field macros defined below.
If these bitfields aren't use, please delete them to simplify this structure.

Reply:
It seems didn't be used in our code.
We will removed these structure in latest commit.